packages/opal-client/opal_client/config.py

```diff
@@ -188,6 +188,8 @@ def load_policy_store():
 
     OPA_HEALTH_CHECK_POLICY_PATH = "opa/healthcheck/opal.rego"
 
+    SCOPE_ID = confi.str("SCOPE_ID", "env", description="OPAL Scope ID")
+
```

Comment:
Should be None maybe so that a client may still use API policy source until we fix it?